core/drivers/cdns_uart.c

```diff
@@ -107,6 +108,7 @@ static const struct serial_ops cdns_uart_ops = {
 	.have_rx_data = cdns_uart_have_rx_data,
 	.putc = cdns_uart_putc,
 };
+KEEP_PAGER(cdns_uart_ops);
```

Comment:
We have a similar situation in `core/arch/arm/kernel/user_ta.c` with `user_ta_ops` and `user_ta_dump_state()`. In order to avoid pulling in unnecessary stuff in unpaged area we're not using `KEEP_PAGER()` directly on the ops variable. Instead we're putting the ops struct in the unpaged area with `__rodata_unpaged` and then `KEEP_PAGER()` on only the callbacks that has to be in the unpaged area.